pkgs/top-level/perl-packages.nix

```diff
@@ -21324,16 +21324,16 @@ let
       url = "mirror://cpan/authors/id/S/SZ/SZBALINT/WWW-Curl-4.17.tar.gz";
       sha256 = "1fmp9aib1kaps9vhs4dwxn7b15kgnlz9f714bxvqsd1j1q8spzsj";
     };
-    patches = [ ../development/perl-modules/WWW-Curl-4.17-Skip-preprocessor-symbol-only-CURL_STRICTER.patch ];
```

Comment:
Hold on, this patch needs to be removed from the nixpkgs tree.

Reply:
Thx again :+1: